packages/e2e-tests/bin/mongosh.js

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env node
+require('../lib/run');
```

Comment:
Doesn’t make sense to keep this file, no?

Reply:
I checked and it seems to be used from inside the test folder: https://github.com/mongodb-js/mongosh/blob/eddb8fb84b4aad6158c4a4a2b41e8cc9943735c6/packages/cli-repl/test/test-shell.ts#L70